modules/exploits/windows/http/easychatserver_seh.rb

```diff
@@ -0,0 +1,61 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::Tcp
+  #include Msf::Exploit::Remote::HttpClient
```

Comment:
You should use the HTTP mixin if possible so you can get all the evasions and stuff.